Resources/doc/reference/configuration/proxy-client.rst

```diff
@@ -80,9 +80,15 @@ nginx
                 servers: 123.123.123.1:6060, 123.123.123.2
                 base_url: yourwebsite.com
                 purge_location: /purge
+        tags:
+            enabled:        false
 
 For ``servers`` and ``base_url``, see above.
 
+.. warning::
+
+    By now, tagging only works with Varnish, so ``tags`` has to be explicitly disabled.
```

Comment:
do we need to explicitly set it to false? it seems we already have this `auto` option, so if its not specified at all, it should already work, no? or is the `auto` behaving incorrectly?